examples/app/src/main/java/com/blazebit/query/app/Main.java

```diff
@@ -254,6 +265,25 @@ private static void testAws(QuerySession session) {
         System.out.println("AwsAccountSummary");
         print(awsAccountSummaryResult);
 
+        TypedQuery<Object[]> awsAccessKeyMetaDataLastUsedQuery = session.createQuery(
+                "select a.* from AwsAccessKeyMetaDataLastUsed a" );
+        List<Object[]> awsAccessKeyMetaDataLastUsed = awsAccessKeyMetaDataLastUsedQuery.getResultList();
+        System.out.println("AwsAccessKeyMetaDataLastUsed");
+        print(awsAccessKeyMetaDataLastUsed);
+
+        // EC2
+        TypedQuery<Object[]> awsInstanceQuery = session.createQuery(
+                "select i.* from AwsInstance i" );
+        List<Object[]> awsInstanceResult = awsInstanceQuery.getResultList();
+        System.out.println("AwsInstances");
+        print(awsInstanceResult);
+
+        TypedQuery<Object[]> awsVolumeQuery = session.createQuery(
+                "select v.* from AwsVolume v" );
+        List<Object[]> awsVolumeResult = awsVolumeQuery.getResultList();
+        System.out.println("AwsVolumes");
+        print(awsVolumeResult);
+
```

Comment:
These changes are good, but the rest needs to be reverted probably. I discussed with @beikov that we are going to add tests here to make this class redundant

Reply:
Ok, reverted those changes